demos/classification_demo/README.md

```diff
@@ -90,6 +90,17 @@ It has the following format:
 ...
 ```
 
+Please note that you should use the ILSVRC 2015 annotation file (val15.txt) with the following models:
+- googlenet-v2
+- se-inception
+- se-resnet-101
+- se-resnet-152
+- se-resnet-50
+- se-resnext-101
+- se-resnext-50
+
+and the ILSVRC 2012 annotation file (val.txt) with all other models supported by the demo.
```

Comment:
You used ground truth term before instead of annotation.

It doesn't make sense to note which file to use for the models without pointing out that it is data set specific and works only if images are taken from imageNet.

You should leave a link to the files where to get them. May be [../../../datasets.md#imagenet](../../../datasets.md#imagenet)

Shouldn't `-labels` file be different for the models as well?